### PR TITLE
candyfloss-shield addition

### PR DESCRIPTION
**Pull request by gavinNBD**
Adding route for Candyfloss on Pokemon Shield

### CODE REVIEW

File: shield-candyfloss/route.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "title": "Candyfloss-Shield", // The name of your route.
```
Comment (Corvimae) [inline shield-candyfloss/route.json]: Remove the comments.

**Review by Corvimae — APPROVED** [review]

File: shield-candyfloss/route.json (hunk 1 above)
Comment (Corvimae) [reply shield-candyfloss/route.json]: also, get rid of the trailing comma after the `version` line.

**Comment by gavinNBD** [comment]
Removed the comments and commas. Not sure if there's any additional action I need to take. (I am not Github savvy.)

**Comment by Corvimae** [comment]
Only the last line shouldn't have a comma, the rest need it.

**Comment by gavinNBD** [comment]
whoops, commas re-added

**Pull request merged**